postcss.config.js

```diff
@@ -34,7 +34,11 @@ module.exports = ({ file, options, env }) => {
   return {
     plugins: [
       require("postcss-bidirection"),
-      require("autoprefixer"),
+      require("autoprefixer")({
+				browsers: ['last 2 Firefox versions', 'last 2 Chrome versions'],
```

Comment:
While we technically only support Firefox and Chrome you could just make this the 'last 2 version' value instead of explicitly just Firefox/Chrome.